temporian/implementation/numpy/data/io.py

```diff
@@ -111,17 +111,17 @@ def event_set(
     if features is None:
         features = {}
 
+    features = {
+        name: normalize_features(value, name)
+        for name, value in features.items()
+    }
+
     # Check timestamps and all features are of same length
     if not all(len(f) == len(timestamps) for f in features.values()):
         raise ValueError(
             "Timestamps and all features must have the same length."
         )
```

Comment:
@achoum would you agree if I move this code into `IndexData` ?

I mean: `normalize_timestamps`, `normalize_features` and check `len(timestamps)==len(features)`